jujugui/static/gui/src/app/app.js

```diff
@@ -1134,6 +1133,7 @@ YUI.add('juju-gui', function(Y) {
           name: cloud
         };
       }
+      const user = this.controllerAPI.get('user');
```

Comment:
Interesting, what puts the user in the connection instances? Should we use `this.controllerAPI.getCredentials().user` instead?